src/pranitaPJun20/BankingSystem.java

```diff
@@ -0,0 +1,76 @@
+package pranitaPJun20;
+
+public class BankingSystem {
+	//Instance Variables
+	int debitCount=0;
+	int creditCount=0;
+	int printBalanceCount=0;
+	//Static variables
+	static int balance=100000;
+	static int staticDebitCount=0;
+	static int staticCreditCount=0;
+	static int staticPrintBalCount=0;
+	
+		//Method to Debit amount
+		void debitAmnt(int amountDebited ){
+			debitCount= debitCount+1;
+			//debitCount++ ; //we can use this also
+			//debitCount+=1 ;//we can use this command also	
+			staticDebitCount=staticDebitCount+1;
+			balance= balance-amountDebited;
+		}
+		//Method to credit amount
+		void creditAmnt(int amountCredited){
+			creditCount= creditCount+1;
+			staticCreditCount=staticCreditCount+1;
+			balance= balance-amountCredited;		
```

Comment:
Do copy-paste code. Here it should be addition operation.

Reply:
Yes.Changed